modules/assignment/src/main/java/com/intuit/wasabi/assignment/impl/AssignmentsImpl.java

```diff
@@ -219,7 +219,7 @@ public Assignment getSingleAssignment(User.ID userID, Application.Name applicati
                     Assignment.Status.EXPERIMENT_EXPIRED);
         }
 
-        Assignment assignment = assignmentsRepository.getAssignment(experimentID, userID, context);
+        Assignment assignment = assignmentsRepository.getAssignment(experimentID, applicationName, userID, context);
```

Comment:
just to test my understanding. applicationName is added because it's need to query experiment_user_index table. Am I correct?